src/app/header/header-profile/header-profile.component.ts

```diff
@@ -30,6 +44,71 @@ export class HeaderProfileComponent implements OnInit, OnDestroy {
     });
   }
 
+  private subscribeToUser(): void {
+    this.authService.user.pipe(
+      takeUntil(this.destroy$)
+    ).subscribe(user => {
```

Comment:
```suggestion
    ).subscribe((user: User) => {
```

Reply:
the same here
![image](https://user-images.githubusercontent.com/64072558/102876078-67d33e80-444d-11eb-91c5-8773c84c81ff.png)

Reply:
The same here :)